### ISSUE TITLE
if the option Allow editing directly in cells is unchecked, nvda does not read anything during cell editing excel

### ISSUE THREAD

**Issue by fernando-jose-silva** [nvaccess/nvda#14544]
### Steps to reproduce:
in excel options, advanced tab uncheck the option
Allow editing directly in cells
now type any value into a cell.
Now move the focus to the cell that received the content earlier and press f2 to edit.
### Actual behavior:
nvda is silent when trying to navigate with arrows or delete characters
### Expected behavior:
nvda should announce the characters or line as appropriate when walking with arrows or deleting characters.
#### NVDA installed/portable/running from source:
instaled
#### NVDA version:
nvda.exe, NVDA alpha-27514,bc164e24

#### Windows version:
11 22623.1095
#### Name and version of other software in use when reproducing the issue:
EXCEL.EXE, Microsoft Office 16.0.15928.20216

#### Does the issue still occur after restarting your computer?
yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
no
#### If NVDA add-ons are disabled, is your problem still occurring?
yes
#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
yes
[excel.txt](https://github.com/nvaccess/nvda/files/10420614/excel.txt)

**Comment by seanbudd** [comment]
Related #4372 #9749

**Comment by Adriani90** [comment]
@fernando-jose-silva are you still having this issue with the last alpha of NVDA or NVDA 2023.1? Did you enable UIA for Excel in advanced settings of NVDA? Does this change anything?

**Comment by fernando-jose-silva** [comment]
playable with the latest stable or alpha version of nvda.
 turning uia on or off for excel makes no difference.
 Thanks.

**Comment by Adriani90** [comment]
I am on Microsoft Excel 2303 Build 16.0.16227.20202), this is the last update which came out today.

**Comment by fernando-jose-silva** [comment]
when navigating between the cells with the arrows, nvda will read the cells normally.

**Comment by fernando-jose-silva** [comment]
sorry, I think I noticed a problem, the option to edit directly in the cells was checked, I unchecked it now and it worked.
everything is with the latest updates.
nvda.exe, NVDA alpha-27981,51aacac6
I checked the updates for office and the latest one for me is:
Microso
ft Office 16.0.16227.20212
anyway we can close, everything working.
Thanks.

**Comment by Adriani90** [comment]
Thanks for the quick update. Closing as works for me.
